org-super-agenda.el

```diff
@@ -713,6 +717,10 @@ e.g. \"A\" or (\"B\" \"C\")."
   :test (cl-member (org-super-agenda--get-priority-cookie item) args :test 'string=))
 
 (cl-defmacro org-super-agenda--defpriority-group (name docstring &key comparator)
+  "Define a priority-grouping function.
+NAME is a symbol appended to `priority' to create the group name.
+DOCSTRING is a string used for the function's docstring.
+COMPARATOR is a grouping comparison operator."
```

Comment:
This should probably say, `COMPARATOR is an unquoted comparator function symbol.`